Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.opensource.zalan.do/library/openjdk-8:latest
+FROM registry.opensource.zalan.do/library/openjdk-11-jre-slim:latest
```

Comment:
Not sure, but maybe `openjdk-11-jdk-slim` is the better option (we anyways won't save a lot of time with such a big installation steps afterwards. 
Plus kafka instances are not recreated every now and then. 
But the jdk tooling may be very useful on kafka instace...

Reply:
Good point.  I was looking at our nakadi base image, but considerations are very different there.